exercises/concept/top-scorers/src/TopScorers.elm

```diff
@@ -0,0 +1,38 @@
+module TopScorers exposing (..)
+
+import TopScorersSupport exposing (PlayerName)
+
+
+updateGoalCountForPlayer : PlayerName -> Dict PlayerName Int -> Dict PlayerName Int
+updateGoalCountForPlayer playerName playerGoalCounts =
+    Debug.todo "implement updateGoalCountForPlayer function, to initialise or increment the goalcount for PlayerName"
```

Comment:
I really like those short explanations, sometimes they are all you need if you want to be fast.